src/index.html

```diff
@@ -11,7 +11,151 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <!-- <h1>HTML Form</h1> -->
     <script type="text/javascript" src="./main.js"></script>
+    <form action=""
+     method="post" onsubmit="onSubmit()">
+      <fieldset class="field">
+        <legend>Personal information</legend>
+
+        <label for="surname" class="form-field">Surname:
+          <input
+           type="text"
+           name="surname"
+           id="surname"
+           autocomplete="off"
+           class="first">
+        </label>
+        <br>
+
+        <label for="name" class="form-field">Name:
+          <input
+           type="text"
+           name="name"
+           id="name" class="first">
+        </label>
+        <br>
+
+        <label for="age" class="form-field">How old are You?
+          <input
+           type="number"
+           name="o-age"
+           id="age"
+           value="12"
+           min="12"
+           max="100"
+           class="first">
+        </label>
+        <br>
+
+        <label for="birth" class="form-field">Full date of birth:
+          <input
+           type="date"
+           name="d-birth"
+           id="birth" class="first">
+        </label>
+        <br>
+
+        <label for="agree">I accept the term of the agreement
+          <input
+           type="checkbox"
+           name="yes"
+           id="agree" class="first">
```

Comment:
```suggestion
          <input
           type="checkbox"
           name="yes"
           id="agree" class="first"
         >
```
закрывающая скобка тега должна быть на одной линии с открывающей